ai/discord.mdx

```diff
@@ -0,0 +1,70 @@
+---
+title: "Discord bot"
+description: "Add a bot to your Discord server that answers questions based on your documentation."
+keywords: ["Discord", "bot", "Q&A"]
+---
+
+<Info>
+  Discord integrations are available on [Pro and Custom plans](https://mintlify.com/pricing?ref=discord) with access to the Mintlify assistant.
+</Info>
+
+Bring the knowledge in your documentation to your Discord server with the Mintlify assistant. The assistant powers a bot that answers user questions with information from your documentation. The bot searches your documentation to provide accurate, cited responses.
```

Comment:
Can we focus it more on the value prop of customer support? i really like the line "the first line of defense for customer support" Im not tied to my suggestion though so im also open to iterations
```suggestion
The Discord bot serves as the first line of defense for customer support. It is trained to answer questions by directly citing the docs and it is always up-to-date.
```